database/ongoing/unsubscribe-users.js

```diff
@@ -0,0 +1,38 @@
+const getItems = require('../utils/get-items');
+const asyncForEach = require('../utils/async-for-each');
+const { logger, generateProgressBar } = require('../utils/logging');
+
+module.exports = {
+  async up(db, client) {
+    const emails = getItems('bouncing-emails.csv');
+    const progress = generateProgressBar(emails.length);
+    await asyncForEach(emails, async (email, index) => {
+      const log = (m, p, t = 'info') => logger(t, index, m, p);
+      log('Query out: ', email);
+      progress.tick();
+      await db
+        .collection('user')
+        .updateOne(
+          { email: email },
+          { $set: { sendQuincyEmail: false } },
+          async (err, result) => {
+            if (err) {
+              log('ERRORED migrating: ', email, 'error');
+              return;
+            }
+            const { modifiedCount } = result;
+            modifiedCount
+              ? log('Success migrating: ', email)
+              : log('Failure migrating: ', email);
```

Comment:
```suggestion
            const { modifiedCount, matchedCount } = result;
            matchedCount && modifiedCount && log('Success migrating: ', email);
            matchedCount
              ? log('Failure updating: ', email)
              : log('Failure finding: ', email);
```
How about this?  Then you'll know if the email was simply missing for some reason.

Reply:
Good point. 

I think this should only have failed if the key being updated is already set at the value it needs to be. i.e `sendQuincyEmail` is already `false`.

But I do not see why these recommended changes would hurt. I will test them out.